templates/launch.html

```diff
@@ -172,6 +99,38 @@ <h1>Launch a survey</h1>
 
 <script>
 
+    function loadMetadata() {
+        var xhttp = new XMLHttpRequest();
+        xhttp.onreadystatechange = function() {
+            if (this.readyState == 4 && this.status == 200) {
+
+                document.getElementById("survey_metadata").innerHTML = ""
+
+                var response = JSON.parse(this.responseText);
+
+                for (var i = 0; i < response.length; i++) {
```

Comment:
If there is no data, it would be good to show something like "No metadata required for this survey" or similar so that it's clear that it's still working

Reply:
👍